### PR TITLE
DAH-1667: Styling Educator Banner

### PR DESCRIPTION
**Pull request by alulabeshue-sfgov**
This PR address the second follow up comment left on [DAH-1667](https://sfgovdt.jira.com/browse/DAH-1667?focusedCommentId=75971), specifically around the spacing between letters. Before opening this PR I sent out some screenshots to Yindi and Bridget to get feedback on the styling in desktop and mobile.

The second PR will be prioritized and handled in [DAH-1715](https://sfgovdt.jira.com/browse/DAH-1715).

Screenshot 2023-09-08 at 1 42 37 PM

Screenshot 2023-09-08 at 1 42 48 PM

[DAH-1667]: https://sfgovdt.jira.com/browse/DAH-1667?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DAH-1715]: https://sfgovdt.jira.com/browse/DAH-1715?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by jimlin-sfgov** [comment]
Is this PR mean to address [additional feedback](https://sfgovdt.jira.com/browse/DAH-1667?focusedCommentId=75971) that was made after the first PR was already merged in?

**Comment by alulabeshue-sfgov** [comment]
> Is this PR meant to address [additional feedback](https://sfgovdt.jira.com/browse/DAH-1667?focusedCommentId=75971) that was made after the first PR was already merged in?

Yes, specifically the second comment around the spacing between letters. This time I sent out a screenshot to Yindi and Bridget to get feedback before opening this PR.

File: app/javascript/modules/listingDetails/ListingDetailsReservedBanner.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.educator1-banner {
+  @apply normal-case;
+  @apply space-y-1;
+  @apply tracking-tight;
+  font-size: 13px;
```
Comment (jimlin-sfgov) [inline app/javascript/modules/listingDetails/ListingDetailsReservedBanner.scss:5]: The convention in our code `scss` files is to use either `rem` or `bloom-font-size` variables to for font sizes.

**Review by jimlin-sfgov — CHANGES_REQUESTED** [review]

**Review by jimlin-sfgov — APPROVED** [review]
looks good!

**Pull request merged**